beast/observationmodel/tests/test_ast_input_list.py

```diff
@@ -0,0 +1,87 @@
+import os.path
+
+import numpy as np
+import h5py
+
+from astropy.utils.data import download_file
+from astropy.tests.helper import remote_data
+
+from ..noisemodel import generic_noisemodel as noisemodel
+from ..noisemodel.absflux_covmat import hst_frac_matrix
+from ...physicsmodel.grid import FileSEDGrid
+from astropy.utils.data import download_file
+from ...observationmodel.ast import make_ast_input_list
+
+
+def _download_rename(filename):
+    """
+    Download a file and rename it to have the right extension
+
+    Otherwise, downloaded file will not have an extension at all
+    """
+    url_loc = 'http://www.stsci.edu/~kgordon/beast/'
+    fname_dld = download_file('%s%s' % (url_loc, filename))
+    extension = filename.split('.')[-1]
+    fname = '%s.%s' % (fname_dld, extension)
+    os.rename(fname_dld, fname)
+    return fname
+
+
+@remote_data
+# @pytest.mark.skip(reason="temporarily disable")
+def test_pick_models():
+<<<<<<< HEAD
```

Comment:
This is causing errors now.  
Usually inserted when there is a conflict with rebasing/merging on your computer.  Needs to be resolved of course before the tests will run at all.